scripts/otel_packaging.py

```diff
@@ -21,14 +21,24 @@
 scripts_path = os.path.dirname(os.path.abspath(__file__))
 root_path = os.path.dirname(scripts_path)
 instrumentations_path = os.path.join(root_path, "instrumentation")
+genai_instrumentations_path = os.path.join(root_path, "instrumentation-genai")
 
 
 def get_instrumentation_packages():
-    for pkg in sorted(os.listdir(instrumentations_path)):
+    pkg_paths = []
+    for pkg in os.listdir(instrumentations_path):
         pkg_path = os.path.join(instrumentations_path, pkg)
         if not os.path.isdir(pkg_path):
             continue
+        pkg_paths.append(pkg_path)
+    for pkg in os.listdir(genai_instrumentations_path):
+        pkg_path = os.path.join(genai_instrumentations_path, pkg)
```

Comment:
nice, I guess this logic bootstraps the necessary packages?

Reply:
@karthikscale3 

Yes! It scans the packages a user has that can be instrumented and installs the relevant instrumentation if exists. Usually the first step before running zero-code mechanisms like `opentelemetry-instrument`.